File: python/1161_soma_de_fatoriais.py
def fatorial(n):
    sum = 1
    for i in range(1,n+1):
        sum = sum*i
    return sum

while True:
    try:
        n1, n2 = map(int, input().split())
        print(fatorial(n1) + fatorial(n2))
    except EOFError:
        break

File: C, C++/1161_soma_de_fatoriais.cpp
#include <iostream>
using namespace std;

long fatorial(int n){
	long long int sum = 1, i;
	for(i = n;i >= 1; i--){
	    sum *= i;
	}
	return sum;
}

int main(){
    int n1, n2;
	while(cin >> n1 >> n2){
		cout << fatorial(n1) + fatorial(n2) << '\n';
	}
    
return 0;
}
